### PR TITLE
STYLES: Updated pages layout

### PR DESCRIPTION
**Pull request by kotesh-arya**

### CODE REVIEW

File: src/pages/ProductList/ProductList.css [hunk 1]
```diff
@@ -1,80 +1,77 @@
...
+}
+
+
+
+.rating-item {
+    list-style-type: none;
+    margin: 1rem 0;
+}
+
+
```
Comment (swethaogeti) [inline src/pages/ProductList/ProductList.css:63]: remove extra spaces!

**Review by swethaogeti — APPROVED** [review]
remove extra spaces and refactor it . Rest is good !

File: src/pages/ProductList/ProductList.css (hunk 1 above)
Comment (kotesh-arya) [reply src/pages/ProductList/ProductList.css:63]: Sure 👍🏻

File: src/pages/HomePage/HomePageComponents/NavBar/NavBar.css [hunk 2]
```diff
@@ -1,87 +1,94 @@
...
 
 .header-input {
-    padding: 13px;
-    width: 90%;
-    border-radius: 3rem;
-    border: none;
+  padding: 9px;
+  width: 90%;
+  border-radius: 3rem;
+  border-radius: 5px;
```
Comment (MaruthiPothuganti) [inline src/pages/HomePage/HomePageComponents/NavBar/NavBar.css:37]: Applied border radius twice

**Review by MaruthiPothuganti — COMMENTED** [review]
Everything looks good with clean code. Try deploying so that a preview can be seen. PR looks good to merge.

File: src/pages/HomePage/HomePageComponents/NavBar/NavBar.css (hunk 2 above)
Comment (kotesh-arya) [reply src/pages/HomePage/HomePageComponents/NavBar/NavBar.css:37]: Corrected it, Thanks for pointing out 👍🏻

**Pull request merged**